problem1.py

```diff
@@ -1 +1,36 @@
 
+"""
```

Comment:
please see my comments in this video.
https://drive.google.com/file/d/1OSQNvQ3TWAnjFuAWYNHRJJjxYROGAIPM/view